pilot/prompts/dev_ops/debug.prompt

```diff
@@ -8,7 +8,6 @@ Ok, we need to debug this issue
 ```
 {{ user_input }}
 ```{% endif -%}
-{{ context }}
```

Comment:
Why is this removed?

If we don't need it anymore we should remove it from Debugger.py where we call the prompt:
```
            llm_response = convo.send_message('dev_ops/debug.prompt',
                {
                    'command': command['command'] if command is not None else None,
                    'user_input': user_input,
                    'issue_description': issue_description,
                    'os': platform.system(),
                    'context': convo.to_context_prompt()
                },
                DEBUG_STEPS_BREAKDOWN)
```

Reply:
This context doesn't give any relevant info to the LLM